Kickstarter-iOS/Views/AmountInputView.swift

```diff
@@ -21,6 +49,10 @@ class AmountInputView: UIView {
 
     _ = ([self.label, self.textField], self.stackView)
       |> ksr_addArrangedSubviewsToStackView()
+
+    self.doneButton.heightAnchor.constraint(equalToConstant: Layout.Button.height).isActive = true
```

Comment:
Should we use `NSLayoutConstraint.activate()` when more than one is being activated?